packages/rest/src/__tests__/acceptance/validation/validation.acceptance.ts

```diff
@@ -160,42 +160,12 @@ describe('Validation at REST level', () => {
 
     before(() =>
       givenAnAppAndAClient(ProductController, {
-        nullable: false,
```

Comment:
I noticed you are removing `nullable` flag in several tests. What is the reasoning for that? Is `nullable` flag no longer recognized? How is it going to affect LB users that are 1) setting `nullable: true` 2) setting `nullable: false`? Can we describe this impact in the release notes please?

Reply:
`nullable` is removed and it's default to `true`. I don't see a way to set it to `false`.